### ISSUE TITLE
Help with BIOMOD_2.6.2.2-Maxent error

### ISSUE THREAD

**Issue by hlstrived** [biomodhub/biomod2#596]
Hello,

I am currently using biomod2 v4.6.2.2 for model parameter setup and single model construction. I have employed four models, including GAM, GLM, RF, and MAXENT. When I pass the configured parameters to the models for execution, MAXENT displays an error. I understand it to be a memory insufficiency issue, but despite adjusting the initial memory settings, the error persists. 
How can this be resolved? 

**Here is my running code:** 

GLMopt <- list(type="quadratic", interaction.level=0, myFormula=NULL,
         family=binomial(link="logit"), test="AIC")

GAMopt <-  list(algo="GAM_mgcv", type="s_smoother", k=4, interaction.level=0, 
           myFormula=NULL, family=binomial(link="logit"))

RFopt <-  list(do.classif=TRUE, ntree=500)

MAXENTopt <- list(path_to_maxent.jar = getwd(),
                  memory_allocated = 6000,
                  initial_heap_size = "default",
                  max_heap_size = "default",
                  background_data_dir = "default",
                  visible = FALSE, 
                  linear = TRUE,   
                  quadratic = TRUE,
                  product = FALSE,  
                  threshold = FALSE,
                  hinge = FALSE,  
                  lq2lqptthreshold = 80,
                  l2lqthreshold = 10,
                  hingethreshold = 15,
                  beta_threshold = -1,
                  beta_categorical = -1,
                  beta_lqp = -1,
                  beta_hinge = -1,
                  betamultiplier = 1,
                  defaultprevalence = 0.5)

create_model_params <- function(opt) {
  list(
    '_allData_allRun' = opt,
    '_PA1_allRun' = opt,
    '_PA1_RUN1' = opt,
    '_PA1_RUN2' = opt,
    '_PA1_RUN3' = opt,
    '_PA1_RUN4' = opt,
    '_PA1_RUN5' = opt
  )
}

user.GAM <- create_model_params(GAMopt)
user.GLM <- create_model_params(GLMopt)
user.MAXENT <- create_model_params(MAXENTopt)
user.RF <- create_model_params(RFopt)

user.val <- list(
  'GAM.binary.mgcv.gam' = user.GAM,
  'GLM.binary.stats.glm' = user.GLM,
  'MAXENT.binary.MAXENT.MAXENT' = user.MAXENT,
  'RF.binary.randomForest.randomForest' = user.RF
)

Model_One <- 
  BIOMOD_Modeling(bm.format = ps_abs,
                  modeling.id = 'AllModels',
                  models = c("GAM", "GLM", "RF","MAXENT"),
                  OPT.strategy = "user.defined",
                  OPT.user.val = user.val,
                  OPT.user.base = "bigboss",
                  CV.strategy = 'random',  
                  CV.nb.rep   = 5,
                  CV.perc     = 0.7,  
                  CV.do.full.models = TRUE, 
                  metric.eval = c('TSS','ROC'),
                  weights = NULL,
                  prevalence= 0.5, 
                  var.import  = 3,
                  nb.cpu = 3,
                  seed.val = NULL,
                  do.progress = TRUE)

**And here is the error result:** although I have only shown the last error message here, all attempts to run MAXENT have failed.

![Image](https://github.com/user-attachments/assets/ead61ee8-9a48-43f5-b947-a09a74a2d557)
**maxent.stderr file:**
Invalid initial heap size: -Xmsdefault
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

In MAXENTopt, I have tried various settings for memory_allocated, including the default value of 512, but all attempts have failed.

Any help would be greatly appreciated.

Best regards,

Ling
